submissions/image_processing/batch_classifier.py

```diff
@@ -0,0 +1,78 @@
+#from rampwf.workflows.image_classifier import get_nb_minibatches
+
+from IPython.display import display
+
+import numpy as np
+
+from skimage import data, color
+from skimage.transform import hough_circle, hough_circle_peaks
+from skimage.feature import canny
+from skimage.draw import circle_perimeter
+from skimage.util import img_as_ubyte
+
+
+class CraterDetector:
+    def __init__(self, sigma=3, threshold=0.4):
+        self.sigma = sigma
+        self.threshold = threshold
+
+    def fit(self, X, y):
+        return self
+
+    def _hough_detection(self, image):
+        edges = canny(image, sigma=self.sigma, low_threshold=10,
+                      high_threshold=50)
+
+        hough_radii = list(np.arange(10, 20, 1)) + list(
+            np.arange(20, 40, 2)) + list(np.arange(40, 100, 4))
+
+
```

Comment:
So in the end we are using the radii from 10 to 100, not 5 to 50 ?

Reply:
ah, good catch. No, I just copy pasted the old, wrong values